docker-bake.hcl

```diff
@@ -80,7 +80,7 @@ target "buildkit-test" {
 target "desktop" {
   inherits = ["mainline"]
   args = {
-    QEMU_PATCHES = "${QEMU_PATCHES},subreaper-prctl,pretcode"
```

Comment:
`subreaper-prctl` patch not necessary anymore?

Reply:
I thought I removed it...

Reply:
Yeah, it should be ok in v9